store/src/java/com/zimbra/cs/redolog/RedoLogManager.java

```diff
@@ -52,6 +53,9 @@
  */
 public class RedoLogManager {
 
+    // if useDbAsLogStorage eq true then will be used a DbLogWriter, FileLogWriter will be used otherwise
+    private boolean useDbAsLogStorage = true;
```

Comment:
Remove this boolean. Synacor will want to switch based on some configuration key and this complicates the future implementation of configuration based dependency injection.

Reply:
ok sure not problem